modules/misc/gtk.nix

```
@@ -154,10 +166,12 @@ in {
     home.packages = optionalPackage cfg.font ++ optionalPackage cfg.theme
       ++ optionalPackage cfg.iconTheme;
 
-    home.file.".gtkrc-2.0".text =
+    home.file.${cfg2.configLocation}.text =
       concatStringsSep "\n" (mapAttrsToList formatGtk2Option ini) + "\n"
       + cfg2.extraConfig;
 
+    home.sessionVariables = { GTK2_RC_FILES = cfg2.configLocation; };
```

Might as well do it without the explicit attribute set:

```suggestion
    home.sessionVariables.GTK2_RC_FILES = cfg2.configLocation;
```